modules/social_features/social_page/social_page.install

```diff
@@ -117,3 +117,20 @@ function social_page_update_8002() {
   // Output logged messages to related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Show thumbnail on File fields when attachment is an image.
+ */
+function social_topic_update_8903() {
```

Comment:
Why did we choose a hook from a different module to be placed in a different module?

Reply:
My bad, naming was incidentally copied incorrectly... 🤦